pages/accountLists/[accountListId]/tools/ToolsWrapper.tsx

```diff
@@ -12,12 +12,14 @@ interface ToolsWrapperProps {
   pageUrl: string;
   selectedMenuId: string; // for later use
   children: ReactElement<unknown, string | JSXElementConstructor<unknown>>;
+  Styles?: React.ReactNode;
```

Comment:
I know I wrote this, but should this have a capital `S`? As `children` is also a component, but it doesn't have a capital `C`.